dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ProcessDefinitionService.java

```diff
@@ -922,7 +922,11 @@ public void importSubProcess(User loginUser, Project targetProject, JSONArray js
                 }
 
                 // check extra params
-                CheckUtils.checkOtherParams(taskNode.getExtras());
+                if(taskNode.getExtras() !=null && !CheckUtils.checkOtherParams(taskNode.getExtras())){
```

Comment:
checkOtherParams has checked null

Reply:
I think null is valid params

Reply:
CheckUtils.checkOtherParams optimized, review please